ci-operator/config/openshift/installer/openshift-installer-release-4.5__rc.yaml

```diff
@@ -0,0 +1,35 @@
+resources:
+  '*':
+    limits: {}
+    requests:
+      cpu: 100m
+      memory: 200Mi
+  baremetal-installer:
+    limits: {}
+    requests:
+      cpu: "3"
+      memory: 5Gi
+  installer:
+    limits: {}
+    requests:
+      cpu: "3"
+      memory: 5Gi
+  installer-artifacts:
+    limits: {}
+    requests:
+      cpu: "3"
+      memory: 5Gi
+  upi-installer:
+    limits: {}
+    requests:
+      cpu: "3"
+      memory: 5Gi
+tag_specification:
+  cluster: https://api.ci.openshift.org
+  name: "4.5"
+  namespace: ocp
+tests:
+- as: e2e-metal-ipi
+  steps:
+    cluster_profile: packet
+    workflow: baremetalds-e2e
```

Comment:
What is `baremetalds`?  I expected `baremetal-ipi-e2e`.

Reply:
It's the name of the step registry workflow for metal IPI.  ds refers to dev-scripts, which is the tooling we use to create the emulated baremetal environment.

The original PR creating it was merged before we got a chance to bikeshed on the name of the registry steps.

Reply:
if you are in the release controller, you job definitions are in openshift/release

Reply:
Re naming, juts make sure there is a follow up renaming the workflow to `metal-ipi-e2e` which is consistent with general naming.